ui/Makefile

```diff
@@ -24,38 +20,38 @@ printsrcs:
 
 lint: deps $(SOURCES)
 	@echo "+ $@"
-	yarn lint
+	npm run lint
 	@touch lint
 
 build: deps $(SOURCES)
 	@echo "+ $@"
-	yarn build
+	npm run build
 	@touch build
 
 .PHONY: start
 start: deps
 	@echo "+ $@"
-	yarn start
+	npm run start
 
 .PHONY: test
 test: deps $(SOURCES)
 	@echo "+ $@"
-	yarn test
+	npm run test
```

Comment:
I have test failures on my machine. This is not something that should be fixed in this PR iff tests pass for you and in GHA CI.

```
$ make test
....
A worker process has failed to exit gracefully and has been force exited. This is likely caused by tests leaking due to improper teardown. Try running with --detectOpenHandles to find leaks. Active timers can also cause this, ensure that .unref() was called on them.
Summary of all failing tests
FAIL src/Containers/Vulnerabilities/ExceptionManagement/components/ExceptionRequestTableCells.test.tsx (1.632 s)
  ● ExceptionRequestTableCells › getRequestedAction › should show the requested action for a pending deferral with CVEs deferred for 30 days

    expect(received).toBe(expected) // Object.is equality

    Expected: "Deferred (in 30 days)"
    Received: "Deferred (in 29 days)"

      212 |             const requestedAction = getRequestedAction(vulnerabilityException, context);
      213 |
    > 214 |             expect(requestedAction).toBe('Deferred (in 30 days)');
          |                                     ^
      215 |         });
      216 |
      217 |         it('should show the requested action for a pending deferral with CVEs deferred indefinitely', () => {

      at Object.toBe (src/Containers/Vulnerabilities/ExceptionManagement/components/ExceptionRequestTableCells.test.tsx:214:37)

  ● ExceptionRequestTableCells › getRequestedAction › should show the requested action for an approved deferral (pending update) with CVEs deferred for 30 days

    expect(received).toBe(expected) // Object.is equality

    Expected: "Deferral pending update (in 30 days)"
    Received: "Deferral pending update (in 29 days)"

      257 |             const requestedAction = getRequestedAction(vulnerabilityException, context);
      258 |
    > 259 |             expect(requestedAction).toBe('Deferral pending update (in 30 days)');
          |                                     ^
      260 |         });
      261 |
      262 |         it('should show the requested action for an approved deferral (pending update) with CVEs deferred indefinitely', () => {

      at Object.toBe (src/Containers/Vulnerabilities/ExceptionManagement/components/ExceptionRequestTableCells.test.tsx:259:37)


Test Suites: 1 failed, 107 passed, 108 total
Tests:       2 failed, 778 passed, 780 total
Snapshots:   33 passed, 33 total
Time:        9.244 s
Ran all test suites.
make: *** [Makefile:39: test] Error 1
```

Reply:
This is slightly concerning, and the timezone difference is my first guess as well. Although if there are test failures because of this it is possible customers might be getting the wrong date as well.

1. Does this still fail for you now, X number of hours later
2. If so, does the same test fail locally against `master`?

Either way, I'll wait to resolve this conversation until we have a ticket to track this on the UI side.

Reply:
First I tried it in the evening. Today I tried it before noon and it also failed.

Switched to `master`, `make clean`+`make test` and the same failures.

Reply:
I took a very quick look at this and noticed that the dates being compared for this test are _exactly_ 30 days apart. Reducing the gap by less than 1 second causes the same failure on my machine. I'm not sure if it is a timezone issue, a locale setting on your machine, or a slightly different version of node but it almost seems as if there is a disagreement somewhere on how the counting should be handled.

I'm pushing an update to this PR that widens the gap between the two dates by one minute, which I hope might fix this issue.

Reply:
No, unfortunately the same outcome on the recent revision (`4.6.x-196-gd461b8e6af`).

Reply:
https://issues.redhat.com/browse/ROX-25753